src/item_use.c

```diff
@@ -263,7 +264,8 @@ void ItemUseOutOfBattle_Bike(u8 taskId)
     }
     else
     {
-        if (Overworld_IsBikingAllowed() == TRUE && IsBikingDisallowedByPlayer() == 0)
+        if (Overworld_IsBikingAllowed() == TRUE && IsBikingDisallowedByPlayer() == 0
+         && FollowerNPCCanBike())
```

Comment:
```suggestion
        if (Overworld_IsBikingAllowed() == TRUE 
         && IsBikingDisallowedByPlayer() == 0
         && FollowerNPCCanBike())
```
once you bring down one check then it makes sense to do it for the whole statement.